src/main/webapp/app/exercises/modeling/manage/modeling-exercise-update.component.html

```diff
@@ -267,6 +267,7 @@ <h3 jhiTranslate="artemisApp.exercise.sections.grading" id="artemisApp.exercise.
             </div>
         }
         @if (!isExamMode) {
+            <jhi-exercise-feedback-suggestion-options [exercise]="modelingExercise" [dueDate]="modelingExercise.dueDate" />
```

Comment:
If you already pass the exercise, why do you need to pass the due date separately as well?

Reply:
The "jhi-exercise-feedback-suggestion-options" options component was implemented with this interface (exercise and due date passed separately) by Max Sölch - judging the the code inside the component it looks like this was a deliberate decision to allow the component to be reactive to the due date being changed. 🤔

`    
ngOnChanges(changes: any) {
        if (!changes['dueDate'].isFirstChange()) {
            if (this.inputControlsDisabled()) {
                this.exercise.feedbackSuggestionModule = this.initialAthenaModule;
            }
        }
    }
`